hail/scripts/deploy.sh

```diff
@@ -26,10 +27,12 @@ REMOTE=$4
 WHEEL=$5
 GITHUB_OAUTH_HEADER_FILE=$6
 HAIL_GENETICS_HAIL_IMAGE=$7
+HAIL_GENETICS_HAILTOP_IMAGE=$7
```

Comment:
Did you intend for this to also be $7?  If you're adding a new parameter, also update all the error lines above to include the new parameter.

I think we might want to change from the current:
```
[[ $1 ]] || err
[[ $2 ]] || err
...
```
to
```
[ $# -eq 10 ] || err
```

Reply:
I guess that doesn't verify all arguments are non-empty. Maybe we should do:
```
[[ -n $0 && -n $1 && -n $2 && ... ]] || err
```

Reply:
It seems a little unnecessary to defend against someone doing `./deploy.sh '' '' '' '' ...`

Reply:
ya.. I honestly feel like `[ $# -eq 10 ] || err` hits a good balance. The boilerplate now feels more footgun-y than the last example you mention

Reply:
as evidenced by my totally messing this up.